gpiozero/boards.py

```diff
@@ -265,7 +265,7 @@ class LEDBarGraph(LEDCollection):
     """
 
     def __init__(self, *pins, **kwargs):
-        super(LEDBarGraph, self).__init__(*pins, pwm=False)
+        super(LEDBarGraph, self).__init__(*pins, **kwargs)
         initial_value = kwargs.get('initial_value', 0)
```

Comment:
Yeah; it's also worth noting that while we use `**kwargs` here we actually document it as `*pins, pwm=False` - so we tell a white lie in the documentation for the sake of clarity.